app/controllers/api/users_controller.rb

```diff
@@ -17,6 +17,16 @@ def create
     end
   end
 
+  def destroy
+    begin
+      user = User.find(params[:id])
+      user.destroy
+      render json: {}, status: :no_content
```

Comment:
Replace lines 22 & 23 with `current_user.destroy` so only the authenticated user can be deleted. Also be sure you want this to be possible as its irreversible!